event-processing/event processor template.template

```diff
@@ -0,0 +1,156 @@
+# TXMA2 audit resources start
+TxMATwoAuditEventProcessorLogGroup:
+  Type: AWS::Logs::LogGroup
+  Properties:
```

Comment:
`event-processing/event processing template.yml` has been replaced by `event-processing/event-processing-template.yml` this can be deleted

Reply:
Done